storage/tree/tile.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tree
+
+// TileID holds an ID of a tile, which is aligned with the tree layout.
+//
+// It assumes that strata heights are multiples of 8, and so the byte
+// representation of the TileID matches NodeID.
+type TileID struct {
+	Root NodeID
+}
+
+// AsKey returns the ID as a string suitable for in-memory mapping.
```

Comment:
Dunno, seems kind of obvious. I don't want to mention revisions in this package, because it's a storage concept. The fixedness of the tree is also easily deferrable from the `TileID` comment ("tre tree") and its members.